YouTubeSpammerPurge.py

```diff
@@ -740,5 +740,5 @@ def confirm_continue(message=""):
     input("\n Press Enter to Exit...")
 
   else:
-    print("\nFinished Executing.")
+    print("\nDone Running The Script.")
```

Comment:
Questionable change

Reply:
> Questionable change

But the only one that won't break anything as far as I can speculate

Reply:
In my opinion, Finished Executing sounds better/more professional than Done Running The Script.

Reply:
Same opinion here